Upstream: No
Reason: restores deprecated GifQuantizeBuffer which some packages (notably libgdiplus) still use

--- a/Makefile
+++ b/Makefile
@@ -95,11 +95,11 @@
 
 $(UTILS):: libgif.a libutil.a
 
-$(LIBGIFSO): $(OBJECTS) $(HEADERS)
+$(LIBGIFSO): $(OBJECTS) $(HEADERS) $(UOBJECTS)
 ifeq ($(UNAME), Darwin)
 	$(CC) $(CFLAGS) -dynamiclib -current_version $(LIBVER) $(OBJECTS) -o $(LIBGIFSO)
 else
-	$(CC) $(CFLAGS) -shared $(LDFLAGS) -Wl,-soname -Wl,$(LIBGIFSOMAJOR) -o $(LIBGIFSO) $(OBJECTS)
+	$(CC) $(CFLAGS) -shared $(LDFLAGS) -Wl,-soname -Wl,$(LIBGIFSOMAJOR) -o $(LIBGIFSO) $(OBJECTS) $(UOBJECTS)
 endif
 
 libgif.a: $(OBJECTS) $(HEADERS)
